utilix/batchq.py

```diff
@@ -27,6 +27,11 @@ def get_server_type():
 
 
 SERVER = get_server_type()
+SINGULARITY_ALIAS = lambda SERVER: {
```

Comment:
I see that pre-commit is complaining about this, is there any chance to modify it?

Reply:
@GiovanniVolta fixed now